playbooks/azure/openshift-cluster/tasks/create_image_from_vm.yml

```diff
@@ -0,0 +1,32 @@
+---
+- azure_rm_virtualmachine:
+    name: vm
+    resource_group: "{{ openshift_azure_resource_group_name }}"
+    allocated: no
+  register: vm
+
+- command: az vm generalize --resource-group "{{ openshift_azure_resource_group_name }}" --name vm
```

Comment:
Not using `name` for tasks is really a bad idea - it's impossible to debug and the performance stats after the playbook run are useless. Names also serve as a comments - what and why is done.

https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#always-name-tasks

Reply:
Indeed; this is already fixed in https://github.com/openshift/openshift-ansible/pull/8121